governance.md

```diff
@@ -0,0 +1,164 @@
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
+Diversity is our strength.
+We have and follow a Code of Conduct.
+  - We currently follow the OpenInfra Foundation Code of Conduct as a member of the OpenInfra Labs.
+3. **Open Design** : We follow a transparent and open process for planning and designing any content, services, and features of Op1st.
+Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+4. **Open Work** : Two of our core tools for successful community building are transparency and accessibility.
+To support that, work and collaboration must be done in public to the greatest degree possible.
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 reviewers and multiple contributions to the project | Op1st GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in a subproject | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to a subproject | [OWNERS] file approver entry|
```

Comment:
Having a special role based on review and approver seems to mean contributions will mostly be around code (or things that could be expressed in the same terms, like docs stored in git, etc). This implicitly send the signal that non-code-like contributions would be less important, as this do not seems to appear in the governance document. 

Could some notes on that be explicitly added ? 

EG, that we focus for now on that type of contribution (eg, git based), but we will be working to extend the governance for others types of contributions based on the activities, or something in that spirit ?

Reply:
I agree with your concern. But I don't have a better solution, yet. Solution in what (content) and how (technical solution)
But maybe this could/should be a topic for v1.1 or later?

Reply:
Reviewing contributions is not limited to code contributions, but maybe this could be called out explicitly?

Reply:
Addressed in 279810a